lib/auth/state_unix.go

```diff
@@ -22,13 +22,18 @@ package auth
 import (
 	"context"
 
+	"github.com/gravitational/teleport/lib/backend"
+	"github.com/gravitational/teleport/lib/backend/kubernetes"
 	"github.com/gravitational/teleport/lib/backend/lite"
-
 	"github.com/gravitational/trace"
 )
 
 // NewProcessStorage returns a new instance of the process storage.
 func NewProcessStorage(ctx context.Context, path string) (*ProcessStorage, error) {
+	var (
+		identityStorage stateBackend
+	)
```

Comment:
```suggestion
	var identityStorage stateBackend = litebk
```

and this should be moved right above the `if kubernetes.InKubeCluster()`, which can then overwrite it if needed